contrib/easyconfig-templates/2023a/PythonBundle-scipy.eb.tmpl

```diff
@@ -0,0 +1,39 @@
+# Template for a python bundle using SciPy-bundle on 2023a
+easyblock = 'PythonBundle'
+
+name = 'SoftwareName'
+version = '0.0.0'
+
+homepage = 'https://www.domain.org'
+description = """
+Description in 80 chars long column
+"""
+
+toolchain = {'name': 'gfbf', 'version': '2023a'}
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('ext-normal', 'ext_version', {
+        'checksums': ['0000000000000000000000000000000000000000000000000000000000000000'],
```

Comment:
usually checksums are added with `--inject-checksums` so this is not really needed

Reply:
right, fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/21984/commits/42561bba065f26f74df737580b21abc6f09ed92f